.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,8 @@
 blank_issues_enabled: true
 contact_links:
-  - name: Gitter chat
-    url: https://gitter.im/tldr-pages/tldr
+  - name: Chatroom
+    url: https://matrix.to/#/#tldr-pages:matrix.org
     about: You can ask and answer questions here
```

Comment:
Seems good to me

Reply:
Let's keep this conversation unresolved so others can offer their thoughts as well.

Reply:
Oxford comma!

I even wrote this a while back too. :sob: 
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#serial-comma

Reply:
I'm fine with adding the comma after "question" :) I don't think that in this case it would make a difference, but it doesn't hurt either.

Btw, I didn't participate in the discussion to add that section, but I wonder if it's really such a prevalent and/or contentious issue that it requires being included in the style guide... :thinking: Besides, the example given there strikes me as odd since the first ambiguous interpretation relies on usage of the comma in a place where there isn't a clause boundary (I'm sure there's a name for that — it's the opposite of a [comma splice](https://en.wikipedia.org/wiki/Comma_splice)) but anyway, both of these observations are topics for a separate discussion.

Reply:
I was going to reply with a different perspective, but honestly, that's just splitting hairs at this point. Let's go with the comma, and avoid derailing this thread to much (if anything, for the sake of who may read this thread in the near or far future — as @kbdharun discovered recently, trawling through long discussions is kind of a drag :sweat_smile:)

Reply:
Sure sure, sorry to be a pain here though. ^-^'

Reply:
Not at all, I actually had to catch myself before going on that tangent — I think it's only natural that people who are attracted to this project share this kind of attention to detail in writing :)